src/Sylius/Bundle/ApiBundle/Provider/LiipProductImageFilterProvider.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Provider;
+
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+/** @experimental */
+class LiipProductImageFilterProvider implements ProductImageFilterProviderInterface
+{
+    use ContainerAwareTrait;
+
+    public function provideAllFilters(): ?array
+    {
+        /** @var array|null $filters */
+        $filters = $this->container->getParameter('liip_imagine.filter_sets');
```

Comment:
Why injecting whole container to get just parameter instead of `__construct()` injection?

Reply:
Also to remove `null` value & prevent fatal mentioned below, it should be:
```suggestion
        $filters = $this->container->getParameter('liip_imagine.filter_sets') ?: [];
```